src/launcher.c

```diff
@@ -769,7 +789,8 @@ int main(int argc, char *argv[])
   case MODEL_NANO_X:
     if (sdk_version != SDK_NANO_X_1_2 && sdk_version != SDK_NANO_X_2_0 &&
         sdk_version != SDK_NANO_X_2_0_2 && sdk_version != SDK_API_LEVEL_1 &&
-        sdk_version != SDK_API_LEVEL_5 && sdk_version != SDK_API_LEVEL_12) {
+        sdk_version != SDK_API_LEVEL_5 && sdk_version != SDK_API_LEVEL_12 &&
+        sdk_version != SDK_API_LEVEL_15) {
```

Comment:
15 doesn't really exist on LNX / LNSP / europa, could be replaced by 18.

Reply:
But it stays valid for Stax?